albumentations/augmentations/transforms.py

```diff
@@ -2725,6 +2725,7 @@ class ToGray(ImageOnlyTransform):
     than 127, invert the resulting grayscale image.
 
     Args:
+        preserve_channels (bool): If True transform return 3 channel image, else 1 channel. Default: True.
```

Comment:
I would make `preserve_channels=False` as a default behavior.

Reply:
I did reset commit. I changed my mind again. 😄 
But I do not sure that is right way, because we provide augmentations and I think they should not change images shapes without explicit directions.
But ToGray transform is mean that that transform give to us 1 channel image... I am little bit confused 😕